students/christopher_gantt/lesson07/html_render.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+class Element(object):
```

Comment:
no need to extend from object in python3

Reply:
To clarify, this means that I can write the class as...   class Element():    ...and not add the object?

Reply:
`class Element:` is python3 style